virtcontainers/network.go

```diff
@@ -598,8 +599,8 @@ const (
 	// NoopNetworkModel is the No-Op network.
 	NoopNetworkModel NetworkModel = "noop"
 
-	// CNMNetworkModel is the CNM network.
-	CNMNetworkModel NetworkModel = "CNM"
+	// DefaultNetworkModel is the default network.
+	DefaultNetworkModel NetworkModel = "default"
```

Comment:
Yeah tell me which name I could use, but I agree "default" is definitely not great...
A bit more background here, initially I wanted to remove the interface since it was not really needed as we now have only one real implementation, but I realized that our unit testing would be way more complicated/impossible? if we decide to remove the noop implementation.

Reply:
@egernst I think naming the network to Default is a bit awkward, lets rework this in a separate PR if required.